docs/rules/jsx-prefer-fragment-shorthand.md

```diff
@@ -0,0 +1,57 @@
+# Enforce standard or shorthand syntax for React fragments (react/jsx-prefer-fragment-shorthand)
+
+In JSX, a React fragment is created either with `<React.Fragment>...</React.Fragment>`, or, using the shorthand syntax, `<>...</>`. This rule allows you to enforce one way or the other.
+
+Support for fragments was added in React v16.2, so the rule will warn on either of these forms if an older React version is specified in [shared settings][shared_settings].
+
+## Rule Options
+
+```js
+...
+"react/jsx-prefer-fragment-shorthand": [<enabled>, <mode>]
+...
+```
+
+### `always` mode
+
+This is the default mode. It will enforce the shorthand syntax for React fragments, with one exception. [Keys or attributes are not supported by the shorthand syntax][short_syntax], so the rule will not warn on standard-syntax fragments that use those.
```

Comment:
What attributes besides “key” are possible here?

Reply:
Currently none, but the docs hint at event handlers in the future: https://reactjs.org/docs/fragments.html#keyed-fragments